### ISSUE TITLE
Change Controls ? 

### ISSUE THREAD

**Issue by djtomcat** [Gistix/portal2vr#38]
Is it possible to change the bindings of the controller ?

**Comment by iam4k33m** [comment]
Yes - [this is a feature native to steamvr. ](https://steamcommunity.com/sharedfiles/filedetails/?id=2029205314)
